Userland/Utilities/imgcmp.cpp

```diff
@@ -37,12 +37,15 @@ static ErrorOr<void> save_image(NonnullRefPtr<Gfx::Bitmap> bitmap, StringView ou
     return Gfx::WebPWriter::encode(*buffered_stream, *bitmap);
 }
 
-static bool are_pixels_equal(Gfx::Color first_pixel, Gfx::Color second_pixel)
+static bool are_pixels_equal(Gfx::Color first_pixel, Gfx::Color second_pixel, int threshold)
```

Comment:
nit: Can we prefer unsigned types for variables that can't be negative? Something like `u8` or `u32` would be better here IMO.